src/olympia/blocklist/mlbf.py

```diff
@@ -61,7 +61,7 @@ def get_blocked_versions(cls):
         return all_versions
 
     @classmethod
-    def get_all_guids(cls, excluding_version_ids=None):
+    def fetch_all_addons_from_db(cls, excluding_version_ids=None):
```

Comment:
The name of the thing being fetched changed from `guids` to `addons`. Is that an improvement? I notice that tests that call this still store the returned value in a variable named, for example, `all_guids`.

Reply:
yeah, I don't know.  It's not strictly just guids, it's a list of `(guid:version)` tuples.  `fetch_all_versions_from_db` might be more accurate.